storage/postgres/src/lib.rs

```diff
@@ -224,21 +264,6 @@ impl StorageCollection for PostgresBucket {
                             return self.set_state(id, state).await; // retry
                         }
                     }
-                    ErrorKind::UndefinedColumn { table, column } => {
```

Comment:
@Aceeri At some point I think we should still detect this error, but just not use the field name from it - we can just use it as a prompt to reload the column cache.

Its not urgent, because this would only happen if a column got removed, but it's a recoverable error, so we should probably handle it.

At a minimum we can invalidate the column cache, even if we don't retry immediately. that way the next attempt can succeed